test/cases/index.js

```diff
@@ -20,6 +20,12 @@ const html = [
     data: minify(read('./complex.html'), {
       collapseWhitespace: true
     })
+  },
+  {
+    name: 'html with script',
+    data: minify(read('./with-script.html'), {
+      collapseWhitespace: true
+    })
```

Comment:
If you rebase/merge `test/server` branch (#26) into your branch and skip this test when the env is jsdom, your test should now pass:

```js
{
  name: 'html with script',
  data: minify(read('./with-script.html'), {
    collapseWhitespace: true
  }),
  get skip() {
    // client parser renders noscript incorrectly in jsdom
    // template renders noscript children as text instead of nodes
    var isJSDOM = typeof window === 'object' && window.name === 'nodejs';
    return isJSDOM;
  }
}
```

Reply:
this seems to work now, do you want me to rebase the test so that it can be merged or should we just close this PR?

Reply:
Since I've added additional test cases, closing the PR should suffice here unless you have any objections.

Reply:
I am good, lets close it.